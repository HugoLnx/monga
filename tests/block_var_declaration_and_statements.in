int main() {
  int[] numbers, tokens;
  char[][] names;

  { int[] numbers, tokens; }
  { char[][] names; }
}
