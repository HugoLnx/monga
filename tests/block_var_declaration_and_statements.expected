declarations:
	declaration:
		function:main,259,0
			block:
				var_declarations:
					variables:
						variable:numbers,259,1
						variable:tokens,259,1
					variables:
						variable:names,260,2
				statements:
					statement:
						block:
							var_declarations:
								variables:
									variable:numbers,259,1
									variable:tokens,259,1
					statement:
						block:
							var_declarations:
								variables:
									variable:names,260,2
					statement:
						attribution:
							var:x
							exp:0,10
					statement:
						attribution:
							var:x
							exp:1,0xa
					statement:
						attribution:
							var:x
							exp:2,j
					statement:
						return:
							exp:0,10
					statement:
						return:
							exp:3,10.30000
					statement:
						return:
							exp:4,"hey jude"
